### ISSUE TITLE
Pamac frequently launches polkit authentication dialog without focus

### ISSUE THREAD

**Issue by geckolinux** [manjaro/pamac#72]
Hi there,

Most times but not all the time, Pamac launches gksu or gksudo (not sure which) without focus, which is a small irritation because you have to click inside the field to type the password.

Thanks for looking into this!

**Comment by philmmanjaro** [comment]
@sb56637: which DE do you use? Which window is in focus instead?

**Comment by sb56637** [comment]
I use Cinnamon. It appears that Pamac itself continues to have focus.

**Comment by philmmanjaro** [comment]
Might be related to that DE then. We mostly use XFCE or Gnome for Pamac on our end. So the password field pops up and if you try to type nothing happens, only after you clicked inside the edit field to get proper focus?

**Comment by sb56637** [comment]
Correct, and the password field's window title bar is also shown as inactive, not focused.

**Comment by guinux** [comment]
Pamac uses polkit not gksu nor gksudo. Your problem is relative to your polkit-authentication-agent and your DE.

**Comment by sb56637** [comment]
I'm not sure about that, since Pamac is the only app that has this focus problem.

**Comment by korrode** [comment]
You've tested other apps that use polkit for authentication?

**Comment by sb56637** [comment]
@korrode  Yep, `system-config-printers` is one that comes to mind that I use frequently.
